_posts/blog/2022-06-22-2.1.0-metrics-and-tracing.md

```diff
@@ -85,7 +85,9 @@ general.micrometer.factory=org.apache.accumulo.test.metrics.TestStatsDRegistryFa
 
 10. Log into Grafana (http://localhost:3003/) using the default credentials (root/root). Click the `Home` icon at the top, then click the `Accumulo Micrometer Test Dashboard`. If everything is working correctly, then you should see something like the image below.
 
-![Grafana Screenshot](/images/blog/202206_metrics_and_tracing/Grafana_Screenshot.png)
+<a class="p-3 border rounded d-block" href="{{ site.baseurl }}/images/blog/202206_metrics_and_tracing/Grafana_Screenshot.png">
+   <img src="{{ site.baseurl }}/images/blog/202206_metrics_and_tracing/Grafana_Screenshot.png" class="img-fluid rounded" alt="Grafana Screenshot"/>
+</a>
```

Comment:
While I was making changes to this file I decided to add the styling we use in other pages for these screenshots. This helps the images remain in line with the other content.

For example:
![image](https://github.com/user-attachments/assets/25aed103-536a-4774-82ac-550a1614e537)

Reply:
It doesn't matter, really, but I think there's probably a way to specify the class on the image or at least on the section (div or p) containing the image, in the markdown form, that might be easier to read or maintain. Markdown, in general, is easier to work with than the raw HTML, but like I said, this doesn't really matter. The raw HTML is fine.